lib/rubocop/cop/rails/three_state_boolean_column.rb

```diff
@@ -60,7 +63,7 @@ def table_node(node)
             node.first_argument
           when :column, :boolean
             ancestor = node.each_ancestor(:block).find do |n|
-              n.method?(:create_table) || n.method?(:change_table)
+              n.method?(:create_table) || n.method?(:change_table) || n.method?(:drop_table) || n.method?(:create_enum)
```

Comment:
`create_enum` was meant to be a representative example of a project-specific wrapper for `create_table`, and I don't think it's appropriate to add project-specific wrappers to a general purpose gem, unless it's via a configuration option.

What does adding `drop_table` do here? Can it take a block?

Reply:
Removed create_enum; though https://blog.saeloun.com/2022/01/12/rails-7-adds-custom-enum-support-in-postgresql is an addition in rails 7. See linked issue for where people are running over it in the real world